lms/djangoapps/ora_staff_grader/mock/data/ora_metadata.json

```diff
@@ -43,25 +44,25 @@
         ],
         "type": "team",
         "rubricConfig": {
-            "feedback_prompt": "How would you grade this response?",
-            "feedback_default_text": "I believe this response...",
+            "feedbackPrompt": "How would you grade this response?",
+            "feedback": "optional",
```

Comment:
Didn't catch this but this is actually an incorrect update. We pass these through from the rubrid config directly which is snake-cased.